util/file_test.go

```diff
@@ -267,5 +281,26 @@ func TestReadFile(t *testing.T) {
 	assert.Equal(t, dataIn, dataOut)
 
 	_, err = ReadFile("/invalid")
-	assert.EqualError(t, err, "open /invalid: no such file or directory")
+	assert.Error(t, err)
+	require.True(t, strings.HasPrefix(err.Error(), "open /invalid: "))
+}
+
+func TestURLStringForPath(t *testing.T) {
+	if runtime.GOOS == "windows" {
```

Comment:
we need to test more complicated paths, like those with weird chars / etc in them